app/views/admin/splits/_details.html.erb

```diff
@@ -0,0 +1,35 @@
+<article class="card">
+   <div class="ArticleHeader">
+      <h2 class="top">Details</h2>
+    </div>
+    <table class="borderless bottom">
+        <tr class="population-row">
+            <td>Population Size</td>
+            <td>
+                <span class='population'><%= split.assignments.count %></span>
+                <span>(<%= link_to "Upload New Assignments", new_admin_split_bulk_assignment_path(split) %>)</span>
+            </td>
+        </tr>
+        <tr>
+            <td>Created At</td>
+            <td><%= split.created_at %></td>
+        </tr>
+        <tr>
+            <td>Finished?</td>
+            <td><%= split.finished? %></td>
+        </tr>
+        <tr>
+            <td>Owner App</td>
+            <td><%= split.owner_app.name %></td>
+        </tr>
+        <tr>
+            <td>&nbsp;</td>
+            <td>
+                <div class="split-decision">
+                    <span class="flag">🏁</span>
+                    <%= link_to "Decide Split", new_admin_split_decision_path(split) %>
+                </div>
+            </td>
+        </tr>
+    </table>
+</article>
```

Comment:
2 spaces for everything except scss is our usual convention. i would just use 2 spaces for the files you added / changed / extracted here. and then we can follow that each time we change something going forward. maybe we will do a separate PR that just re-indents everything later.